app/views/certificates/secondary_certificate/show.html.erb

```diff
@@ -2,7 +2,7 @@
 <% meta_tag :url, request.original_url %>
 <%= render 'pages/certification/hero' %>
 <div class="ncce-programmes-activity">
-  <%= render ProgressBarComponent.new(current_user, @programme) %>
+  <% render ProgressBarComponent.new(current_user, @programme) %>
```

Comment:
pls put back the equals sign, unless you intended to hide your work from the secondary certificate

Reply:
resolved, thank you